Gems/ROS2/Code/Source/Camera/ROS2CameraSensorComponent.cpp

```diff
@@ -100,9 +115,9 @@ namespace ROS2
         const AZ::Transform transform = GetEntity()->GetTransform()->GetWorldTM();
         const auto timestamp = ROS2Interface::Get()->GetROSTimestamp();
         std_msgs::msg::Header ros_header;
-        if (!m_cameraSensorsWithPublihsers.empty())
+        if (!m_imagePublishers.empty() && m_cameraSensor)
         {
-            const auto& camera_descritpion = m_cameraSensorsWithPublihsers.front().second->GetCameraSensorDescription();
+            const auto& camera_descritpion = m_cameraSensor->GetCameraSensorDescription();
```

Comment:
```suggestion
            const auto& camera_description = m_cameraSensor->GetCameraSensorDescription();
```

Reply:
Also the variable naming should be lowerCamelCase